R/binomial.R

```diff
@@ -159,3 +159,51 @@ quantile.binomial <- function(d, p, ...) {
 
   qbinom(p = p, size = d$size, prob = d$p)
 }
+
+#' Fit a binomial distribution to data
+#'
+#' The fit distribution will inherit the same `size` parameter as
+#' the `binomial` object passed.
+#'
+#' @param d A `binomial` object.
+#' @param x A vector of zeroes and ones to fit the binomial distribution to.
+#'
+#' @return a `binomial` object
+#' @export
+fit_mle.binomial <- function(d, x, ...) {
+  ss <- suff_stat(d, x, ...)
+  binomial(ss$trials, ss$successes / (ss$experiments * ss$trials))
+}
+
+#' Compute the sufficient statistics for the binomial distribution from data
+#'
```

Comment:
It would be good to document the `@return` for all the `suff_stat()` methods. This should have two parts: 1. that the return is a list, and the names of the elements of the list, and 2. adding some notes on the sufficient statistics to the original `@details` in the `bernoulli` object documentation.

Reply:
Each `suff_stat` method's `@return` is now documented. I haven't gotten to adding documentation to the `@details` yet.